velox/dwio/common/Statistics.h

```diff
@@ -542,12 +542,18 @@ struct RuntimeStatistics {
   // Number of splits skipped based on statistics.
   int64_t skippedSplits{0};
 
+  // Number of splits processed based on statistics.
+  int64_t processedSplits{0};
+
   // Total bytes in splits skipped based on statistics.
   int64_t skippedSplitBytes{0};
 
   // Number of strides (row groups) skipped based on statistics.
   int64_t skippedStrides{0};
 
+  // Number of strides (row groups) processed based on statistics.
+  int64_t processedStrides{0};
```

Comment:
I think you probably meant "stripe"?

Reply:
Thanks for your review. I follow the `skippedStrides`, and shall we change them?

Reply:
It is stride here but not split